### DISCUSSION TITLE
New Alternative

### DISCUSSION THREAD

**Discussion by Callidus2000** [FortiPower/PowerFMG#9] [General]
Hi,
if you need additional access maybe my module may help:

https://github.com/Callidus2000/FortiManager/

**Comment by alagoutte** [comment]
Hi @Callidus2000 

Nice Thanks ! (i convert to discussions)

i work actually on PowerFGT ;-) with directly fortigate !

**Reply by Callidus2000** [reply]
Hi,
fine for me. Sorry for creating an issue but either I've overseen the discussion tab (man, I'm getting old) or it was not there at this time.

Would you mind if I take inspiration (aka copy'n'paste :smile:) from your readme? Many design principles are the same in both modules but reading through your main readme urges me to update mine...

**Reply by alagoutte** [reply]
> Hi, fine for me. Sorry for creating an issue but either I've overseen the discussion tab (man, I'm getting old) or it was not there at this time.
Yes, it is me, i have add the tab

> Would you mind if I take inspiration (aka copy'n'paste 😄) from your readme? Many design principles are the same in both modules but reading through your main readme urges me to update mine...
Yes, you can copy/paste
